application/src/main/resources/ecc.yml

```diff
@@ -27,16 +27,23 @@ connection:
     host: localhost
     port: 9042
     ##
-    ## Connection Timeout for CQL.
+    ## Connection Timeout for an CQL attempt.
     ## Specify a time to wait for cassandra to come up.
-    ## Connection is tried every five second until either the timeout time passes or the connection is successful.
+    ## Connection is tried based on retry policy delay calculations. Each connection attempt will use timeout to calculate CQL Connection proccess delay.
```

Comment:
"will use timeout" -> "will use the timeout"
Connection -> connection
proccess -> process